src/parser/jobs/ast/modules/Tincture.tsx

```diff
@@ -4,8 +4,16 @@ import {ExpectedActionsEvaluator} from 'parser/core/modules/ActionWindow'
 import {SEVERITY} from 'parser/core/modules/Suggestions'
 import {Tincture as CoreTincture} from 'parser/core/modules/Tincture'
 import React from 'react'
+import {Message} from 'semantic-ui-react'
 
 export class Tincture extends CoreTincture {
+
+	override prependMessages = <Message>
+		<Trans id="sch.tincture.prepend.content">
```

Comment:
Gotta modify the i18n key here for ast

Reply:
```suggestion
		<Trans id="ast.tincture.prepend.content">
```

Reply:
I may be the dumbest man to ever commit to this repo LMAO